metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures ntp as a client or server'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '1.7.0'
+version           '1.8.0'
```

Comment:
You should probably let the author bump the version when appropriate to fit their release plans